arch-design/multicloud/Multi_Cloud_HA.md

```diff
@@ -0,0 +1,65 @@
+# Multi-Cloud High Availability Design
```

Comment:
Will these changes will have any impact on the individual services of multi-cloud?

Reply:
No, hopefully not.